vsts-fullbuild-ubuntu-vm/azuredeploy.json

```diff
@@ -0,0 +1,216 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vstsAccountURL": {
+      "type": "string",
+      "metadata": {
+        "description": "Team Services account URL"
+      }
+    },
+    "vstsPAT": {
+      "type": "string",
+      "metadata": {
+        "description": "Team Services PAT for user with build permissions"
+      }
+    },
+    "vstsPoolName": {
+      "type": "string",
+      "defaultValue": "default",
+      "metadata": {
+        "description": "Team Services Agent Pool Name"
+      }
+    },
+    "vstsAgentName": {
+      "type": "string",
+      "defaultValue": "fullbuild-ubuntu-auto",
+      "metadata": {
+        "description": "Team Services Agent Name"
+      }
+    },    
+    "newStorageAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Storage Account"
+      }
+    },
+    "adminUserName": {
+      "type": "string"
+    },
+    "adminPassword": {
+      "type": "securestring"
+    },
+    "imagePublisher": {
+      "type": "string",
+      "defaultValue": "canonical",
+      "metadata": {
+        "description": "Image Publisher"
+      }
+    },
+    "imageOffer": {
+      "type": "string",
+      "defaultValue": "ubuntuserver",
+      "metadata": {
+        "description": "Image Offer"
+      }
+    },
+    "imageSKU": {
+      "type": "string",
+      "defaultValue": "16.04.0-LTS",
+      "metadata": {
+        "description": "Image SKU"
+      }
+    },
+    "vmDnsName": {
+      "type": "string"
+    }
+  },
+  "variables": {
+    "nicName": "[concat('myNic1',uniqueString(resourceGroup().id, deployment().name))]",
+    "addressPrefix": "10.0.0.0/16",
+    "subnet1Name": "Subnet-1",
+    "subnet1Prefix": "10.0.0.0/24",
+    "publicIPAddressName": "[concat('myIP1',uniqueString(resourceGroup().id, deployment().name))]",
+    "publicIPAddressType": "Dynamic",
+    "vmStorageAccountContainerName": "vhds",
+    "vmName": "[parameters('vmDnsName')]",
+    "vmSize": "Standard_A0",
+    "virtualNetworkName": "MyVNET",
+    "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+    "storageAccountType": "Standard_LRS",
+    "subnet1Ref": "[concat(variables('vnetID'),'/subnets/',variables('subnet1Name'))]"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Storage/storageAccounts",
+      "name": "[parameters('newStorageAccountName')]",
+      "apiVersion": "2015-05-01-preview",
```

Comment:
Use new storage API and properties. See https://github.com/Azure/azure-quickstart-templates/blob/master/101-storage-account-create/azuredeploy.json#L24